contracts/AnchoredPriceView.sol

```diff
@@ -46,7 +46,7 @@ contract AnchoredPriceView {
     uint256 lowerBoundAnchorRatio;
 
     /// @notice The mapping of posted by source prices per symbol
-    mapping(string => uint64) _prices;
+    mapping(string => uint64) public _prices;
```

Comment:
What's the benefit of `uint64` since these don't really save gas?

Reply:
I am good to go with 256